CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+- [FEATURE] Add ability to refresh dashboard #777
```

Comment:
There is also a breaking change since the plugin-system timeRange type is TimeRangeValue now, for ex our internal topology view plugin expects useTimeRange's timeRange to always be absolute. It will be a quick fix though, I love how much this refactor simplifies things 💥